GDJS/tests/tests/spriteruntimeobject.js

```diff
@@ -1,41 +1,141 @@
 // @ts-check
 
-/**
- * Basic tests for gdjs.SpriteRuntimeObject
- */
-describe('gdjs.SpriteRuntimeObject', function() {
-  var runtimeGame = new gdjs.RuntimeGame({
+const createObjectWithAnimationInScene = (runtimeScene) => {
```

Comment:
Put this inside the `describe` (otherwise it will go to the global namespace, the tests run in Karma are just injected like plain old javascript files without being modules)